docs/en/operations/table_engines/mergetree.md

```diff
@@ -313,6 +313,17 @@ INDEX sample_index2 (u64 * length(str), i32 + f64 * 100, date, str) TYPE set(100
 INDEX sample_index3 (lower(str), str) TYPE ngrambf_v1(3, 256, 2, 0) GRANULARITY 4
 ```
 
+#### Functions Support
+
+
+Comparison and search functions can use different indexes instead of the full scan of the table during the query processing. All the functions can use primary key index. Some of the functions can use data skipping indexes.
```

Comment:
> Comparison and search functions can use different indexes instead of the full scan of the table during the query processing.

Functions do not use any index. ClickHouse can use index to perform query if there's a WHERE condition involving the supported subset of function calls (which is supposed to be described here) over columns that are part of index.

> All the functions can use primary key index.

Afaik that's not really true. For example, ngramDistance can't.